src/handlers/interventions-processor-handler.ts

```diff
@@ -154,12 +160,15 @@ async function handleError(error: unknown, record: SQSRecord) {
  */
 function getEventName(recordBody: TxMAIngressEvent): EventsEnum {
   logger.debug('event is valid, starting processing');
-  if (recordBody.event_name === TriggerEventsEnum.TICF_ACCOUNT_INTERVENTION) {
+  if (
+    recordBody.event_name === TriggerEventsEnum.TICF_ACCOUNT_INTERVENTION ||
+    recordBody.event_name === TriggerEventsEnum.OPERATIONAL_ACCOUNT_INTERVENTION
+  ) {
     validateInterventionEvent(recordBody);
     const interventionCode = recordBody.extensions!.intervention!.intervention_code;
     return accountStateEngine.getInterventionEnumFromCode(interventionCode);
   }
-  return recordBody.event_name as unknown as EventsEnum;
+  return recordBody.event_name.toString() as EventsEnum;
```

Comment:
is there an advantage in doing this? event_name will already be a string, so is this operation neeeded?

Reply:
we can either use .toString() or 
`recordBody.event_name as unknown as EventsEnum`
The compiler is complaining otherwise 
`Conversion of type TriggerEventsEnum to type EventsEnum may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to unknown first`

Reply:
yeah exactly, maybe toString() is easier to read then doing `unknown` but it is a method call that we can avoid by just doing that explicit type declaration for the compiler :)

Reply:
I'll loop in @KG700 here for reference. I can't remember why we wanted to call `.toString()` instead of casting to `unknown`?